linode/resource_linode_object_storage_bucket.go

```diff
@@ -60,17 +72,24 @@ func resourceLinodeObjectStorageBucketRead(d *schema.ResourceData, meta interfac
 	client := meta.(*ProviderMeta).Client
 	cluster, label, err := decodeLinodeObjectStorageBucketID(d.Id())
 	if err != nil {
-		return fmt.Errorf("Error parsing Linode ObjectStorageBucket id %s", d.Id())
+		return fmt.Errorf("failed to parse Linode ObjectStorageBucket id %s", d.Id())
```

Comment:
make sure some reference to error message changes makes it into the changelog on release, in case someone is grepping for the old text.